elro/auth.py

```diff
@@ -76,6 +82,31 @@ def session(self) -> ElroConnectsCloudSessionCache | None:
         """Return the current session."""
         return self._session_cache
 
+    def _get_domain(self):
+        """Return the API main domain name address"""
+
+        sckt = socket.socket(socket.AF_INET,socket.SOCK_STREAM)
+        host="info.hekr.me"
+        port=91
+        sckt.connect((host,port))
+        message = "{\"action\":\"getAppDomain\"}\n"
+        sckt.send(message.encode())
+        msg = sckt.recv(1024)
+
+        try:
+            parsed_data = json.loads(msg)
+        except json.decoder.JSONDecodeError:
+            return ""
```

Comment:
May be return "hekr.me" here. We could also return `None`.

Reply:
Changed the way _get_domain is used, see https://github.com/jbouwh/lib-elro-connects/pull/16#discussion_r947232997